scripts/analysis/.gitignore

```diff
@@ -0,0 +1,4 @@
+config.json
```

Comment:
does this need to live nested here? if we pull it all up to the top level `gitignore`, it's easier to discover what's hidden where.

you can always use the full path to scope it to this dir if we need to

Reply:
Done in https://github.com/guardian/source/pull/972/commits/30305b6a42724d715c545e46c13e62450efff73a